control-plane/csi-driver/src/bin/node/mount.rs

```diff
@@ -425,3 +426,48 @@ pub(crate) async fn unmount_on_fs_id_diff(
         )
     })
 }
+
+pub(crate) async fn lazy_unmount_mountpaths(mountpaths: &Vec<DeviceMount>) -> Result<(), Status> {
+    for mountpath in mountpaths {
+        debug!(
+            "Unmounting path: {}, with DETACH flag",
+            mountpath.mount_path()
+        );
+        let target_path = mountpath.mount_path().to_string();
+
+        runtime::spawn_blocking({
+            let target_path = target_path.clone();
+            move || {
+                let mut unmount_flags = UnmountFlags::empty();
+                unmount_flags.insert(UnmountFlags::DETACH);
+                sys_mount::unmount(target_path, unmount_flags)
+                    .map_err(|error| Status::aborted(error.to_string()))
+            }
+        })
+        .await
+        .map_err(|error| Status::aborted(error.to_string()))??;
+
+        if target_path.ends_with("globalmount") {
```

Comment:
Since this didn't work, we can let it be?
Btw did you test cleaning up the entire path?

Reply:
Yeah tried that as well. I will remove this for now.